.github/workflows/check-catalyst.yaml

```diff
@@ -400,6 +398,7 @@ jobs:
     name: Frontend Tests (backend="lightning.kokkos")
     needs: [constants, llvm, runtime, quantum]
     runs-on: ubuntu-latest
+    if: false
```

Comment:
```suggestion
```

Reply:
This needs to be here if we don't build the runtime with kokkos